Vagrantfile

```diff
@@ -0,0 +1,26 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "centos/7"
+  config.vm.synced_folder ".", "/vagrant", disabled: true
+
+  config.vm.provision "shell", inline: <<-SHELL
+    yum -y install epel-release
+    yum -y install puppet5-release || yum -y install https://yum.puppetlabs.com/puppet5/puppet5-release-el-7.noarch.rpm
```

Comment:
when would the first part succeed?

Reply:
On the second run. I often did a `vagrant rsync && vagrant provision` to apply changes.

Reply:
ah, fair